extras/indexing/src/test/java/org/apache/rya/indexing/mongo/MongoFreeTextIndexerTest.java

```diff
@@ -43,27 +44,36 @@
 import org.openrdf.model.vocabulary.RDFS;
 
 import com.google.common.collect.Sets;
+import com.mongodb.MongoClient;
 
 import info.aduna.iteration.CloseableIteration;
 
-public class MongoFreeTextIndexerTest extends MongoRyaTestBase {
+public class MongoFreeTextIndexerTest {
```

Comment:
This isn't really an integration test.  An integration test would create a sail connection and ensure the indexer was being used from start to finish. A lot of the indexers are missing integration tests